animation_nodes/utils/bvh.pyx

```diff
@@ -0,0 +1,29 @@
+from random import random
+from mathutils import Vector
+
+cdef float hitsPerLocation(bvhTree, vector):
+    direction1 = Vector((random(), random(), random())).normalized()
```

Comment:
Move those variables into global variables like in `is_inside_volume.py`. Also, call the function from the util file in the `is_inside_volume.py` file. The main reason we did that was to avoid duplicate code.